File: beginner/pagina12/3209/3209.py
n = int(input())

for _ in range(n):
    reguas = [int(el) for el in input().split()][1:]
    total = reguas[0]

    for i in range(1, len(reguas)):
        total = (total - 1) + reguas[i]

    print(total)

File: beginner/pagina12/3209/3209.php
<?php 

    $n = (int) readline();

    for($i = 0; $i < $n; $i++){
        $data = explode(" ", readline());

        $t = (int) $data[0];
        $reguas = array_map('intval', array_slice($data, 1));

        $total = $reguas[0];

        for($j = 0; $j < $t; $j++){
            $total = ($total - 1) + $reguas[$j];
        }
        echo "$total\n";
    }
?>